Makefile

```diff
@@ -60,6 +60,12 @@ trypush: check-in-vagrant build-clang-plugin build-rust-tools
 	/vagrant/infrastructure/web-server-setup.sh ~/mozilla-config trypush.json ~/trypush-index ~
 	/vagrant/infrastructure/web-server-run.sh ~/mozilla-config ~/trypush-index ~
 
+nss-reblame: check-in-vagrant build-rust-tools
+	[ -d ~/mozilla-config ] || git clone https://github.com/mozsearch/mozsearch-mozilla ~/mozilla-config
```

Comment:
I've never been happier or sadder to know that `[` is actually a command!  (Which is to say, I yearn for the simpler days when I would have gone: "what on earth is going on here?!?".  This makes sense though and is something it makes sense to presume contributors will learn quickly if they don't already know.)